soh/soh/resource/importer/AudioSequenceFactory.cpp

```diff
@@ -15,7 +15,7 @@ ResourceFactoryBinaryAudioSequenceV2::ReadResource(std::shared_ptr<Ship::File> f
 
     audioSequence->sequence.seqDataSize = reader->ReadInt32();
     audioSequence->sequenceData.reserve(audioSequence->sequence.seqDataSize);
-    for (uint32_t i = 0; i < audioSequence->sequence.seqDataSize; i++) {
+    for (int32_t i = 0; i < audioSequence->sequence.seqDataSize; i++) {
```

Comment:
feels a little odd using signed for the counter in a loop but i get that it's just because we read `seqDataSize` as signed. part of me thinks we should check to make sure `seqDataSize` is invalid (`<= 0`) and early return if it is, then cast, but that feels like a riskier change than this so using signed for the counter is probably the best choice here.